src/std_extensions/logic.rs

```diff
@@ -171,6 +187,33 @@ impl MakeRegisteredOp for NotOp {
     }
 }
 
+fn read_inputs(consts: &[(IncomingPort, ops::Const)]) -> Option<Vec<bool>> {
+    let true_val = ops::Const::true_val();
+    let false_val = ops::Const::false_val();
+    let inps: Option<Vec<bool>> = sorted_consts(consts)
+        .into_iter()
+        .map(|c| {
+            if c == &true_val {
+                Some(true)
+            } else if c == &false_val {
+                Some(false)
+            } else {
+                None
```

Comment:
This must be correct (since it compiles), but it looks like it's making a vector of optional bools rather than an optional vector of bools... what am I missing?

Reply:
Ah OK!